SokolTextGame/Commands.cs

```diff
@@ -46,15 +47,67 @@ public void Execute(string[] words, World world)
         {
             if (string.Join(" ", words) == "where") Console.Write("Where? Where what? Or where who?\n");
             else if (string.Join(" ", words) == "where am i") Console.Write(world.CurrentLocation().Description);
-            else Console.Write("I know the command \"where am i\"\n");
+            else if (string.Join(" ", words) == "where can i go")
+            {
+                string output = string.Join("", world.CurrentLocation().possibleLocation);
+                Console.ForegroundColor = ConsoleColor.Green;
+                Console.WriteLine($"Possible location : {output}");
+                Console.ResetColor();
+            }
+            else Console.Write("I know the command \"where am i\" and \"where can i go\"\n");
+        }
+    }
+    public class GoTo : ICommand
+    {
+        public void Execute(string[] words, World world)
+        {
+            if (words.Length == 3)
+            {
+                if (string.Join(" ", words.Take(2)) == "go to")
+                {
+                    string locationName = string.Join(" ", words.Skip(2));
+                    if (world?.player?.CurrentLocation == locationName)
+                    {
+                        Console.ForegroundColor = ConsoleColor.Yellow;
+                        Console.WriteLine($"I am already in a location {world.player.CurrentLocation}");
+                        Console.ResetColor();
+                    }
+                    else
+                    {
+                        var aviableLocation = world.CurrentLocation();
+                        if (aviableLocation.possibleLocation.Contains(locationName))
+                        {
+                            world.player.CurrentLocation = locationName;
+                            Console.ForegroundColor = ConsoleColor.Yellow;
+                            Console.WriteLine($"Current location: {locationName.ToUpper()}");
+                            Console.ResetColor();
+                        }
+                    }
+                }
+            }
+            else if (string.Join(" ", words) == "go")
+            {
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine("Where are we going?");
+                Console.ResetColor();
+            }
+            else if (string.Join(" ", words) == "go to")
+            {
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine("Not bad, not bad.\n" +
+                    "But you have to give the name of the place you want to go to.");
+                Console.ResetColor();
+            }
```

Comment:
You check if user said exactly `go` or exactly `go to`, but in case if they said something completely incorrect (example `go have fun`), the game doesn't react at all, similar to `go to KEKW`. This case has to be handled too.